app/src/main/java/com/kickstarter/ui/viewholders/SurveyViewHolder.kt

```diff
@@ -50,20 +53,25 @@ class SurveyViewHolder(private val binding: ActivitySurveyViewBinding) :
 
     init {
         viewModel.outputs.creatorAvatarImageUrl()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .compose(Transformers.observeForUIV2())
             .subscribe { setCreatorAvatarImage(it) }
+            .addToDisposable(disposables)
         viewModel.outputs.creatorNameTextViewText()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .compose(Transformers.observeForUIV2())
             .subscribe { binding.surveyTitle.text = it }
+            .addToDisposable(disposables)
         viewModel.outputs.projectForSurveyDescription()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .compose(Transformers.observeForUIV2())
             .subscribe { setSurveyDescription(it) }
+            .addToDisposable(disposables)
         viewModel.outputs.startSurveyResponseActivity()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .compose(Transformers.observeForUIV2())
             .subscribe { startSurveyResponseActivity(it) }
+            .addToDisposable(disposables)
+    }
+
+    override fun destroy() {
+        disposables.clear()
```

Comment:
we also need the clear the viewmodel subscriptions here, since we are not inheriting from android `ViewModel`

Reply:
the viewmodel doesn't have any disposables to clear

Reply:
ohp you right!